src/tree_walker.js

```diff
@@ -39,7 +39,8 @@ function TreeWalker(node) {
 
 
 /**
- * @return {!Node} The current parent of the current location in the subtree.
+ * @return {!Element|!DocumentFragment} The current parent of the current
+ *                                      location in the subtree.
```

Comment:
The continuation line should be indented 4 spaces from the start (see http://google.github.io/styleguide/javascriptguide.xml#Comments)